src/Components/Table/ColumnSelection/LogsTableNavField.tsx

```diff
@@ -73,6 +79,19 @@ export function LogsTableNavField(props: {
               </div>
             </div>
           )}
+          {props.columnWidthMap && props.setColumnWidthMap && props.columnWidthMap?.[props.label] !== undefined && (
+            <div
+              onClick={() => {
+                const { [props.label]: omit, ...map } = { ...props.columnWidthMap };
+                props?.setColumnWidthMap?.(map);
```

Comment:
```suggestion
                props.setColumnWidthMap(map);
```

Reply:
This throws a type error, we'd need to add an additional `typeof props.setColumnWidthMap === 'function'` conditional outside of the react template conditional, i.e. 
```
if(typeof props.setColumnWidthMap === 'function'){
  props.setColumnWidthMap(map);
}
```

Reply:
I was able to remove the first optional chaining operator however, just not the for the function call

Reply:
Yeah, it's super annoying. Thanks for checking!